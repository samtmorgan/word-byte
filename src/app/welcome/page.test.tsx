import React from 'react';
import { render } from '@testing-library/react';
import Welcome from './page';

describe('Welcome Page', () => {
  it('renders correctly', () => {
    const { container } = render(<Welcome />);
    expect(container).toMatchSnapshot();
  });
});
