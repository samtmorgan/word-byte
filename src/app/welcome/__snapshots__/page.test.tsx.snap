// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      Welcome Back!
    </h1>
  </div>
</div>
`;
